data/resources/style.css

```diff
@@ -243,6 +243,25 @@ messagebubble.document.outgoing .file > overlay > statusindicator {
   color: #79c271;
 }
 
+messagebubble.document .file loadingindicator {
+  margin: 2px;
+  color: @window_bg_color;
+}
+
+messagebubble.document.outgoing .file loadingindicator {
+  color: @accent_bg_color;
+}
+
+messagebubble.document .file.with-thumbnail > overlay > image {
+  background: alpha(black, 0.6);
+  color: white;
+}
+
+messagebubble.document .file.with-thumbnail loadingindicator {
+  margin: 12px;
+  color: white;
+}
+
```

Comment:
I think that styles need changing too.  Probably that will be enough, because everything else should be handled by `statusindicator`
```suggestion
statusindicator > loadingindicator {
  margin: 2px;
}
```

Reply:
done